File: fib.py
import time 
MAX_N = 50

def fib(n):
    if n <= 1:
        return n 
    return fib(n-1) + fib(n-2)

num = int(input("Enter a number to calculate fib(n) in python: "))
while (num > MAX_N or num < 0):
    num = int(input("Enter an integer 0 <= n <= {MAX_N}: "))

start_time = time.time()
print(fib(num))
print(f"Seconds Taken: {time.time()-start_time}")

File: fib.c
#include <stdio.h>
#include <time.h>

int fib(int n) {
    if ( n <= 1) {
        return n;
    } 
    return fib(n-1) + fib(n-2);
}

int main() {
    int num;
    printf("Enter a number to calculate fib(n) in C: ");
    scanf("%d", &num); 

    clock_t t = clock();
    printf("%d\n", fib(num));
    double time_taken = ((double) clock() - t) / CLOCKS_PER_SEC;
    
    printf("Seconds Taken: %lf\n", time_taken); 
}
